### PR TITLE
Qa/e 905

### PR DESCRIPTION
**Pull request by haydencleary**

### CODE REVIEW

File: src/components/New/Field/Datepicker.tsx [hunk 1]
```diff
@@ -56,19 +82,47 @@ export function Datepicker({ isElevated = false, ...props }: SingleProps | Range
     return Boolean(parseDateString(props.value));
   }, [props.mode, props.value]);
 
+  // If a period is selected use the period's value, otherwise use the custom value
+  const selectedValue = useMemo(() => {
+    return props.selectedPeriod !== Period.Custom && props.periods
```
Comment (alexbeno) [inline src/components/New/Field/Datepicker.tsx:87]: if and return for better readability ?

Comment (alexbeno) [reply src/components/New/Field/Datepicker.tsx:87]: like line 94

File: src/components/New/Filter/useDatepickerPeriods.ts
Comment (alexbeno) [inline src/components/New/Filter/useDatepickerPeriods.ts:1]: don't know if we want to do it now but maybe we can do 

/Filter/FilterDatepicker/
           FilterDatepicker.tsx
           FilterDatepicker.hooks.ts

File: src/components/New/Field/Datepicker.tsx (hunk 1 above)
Comment (AnthonyBuisset) [reply src/components/New/Field/Datepicker.tsx:87]: good catch

File: src/components/New/Filter/useDatepickerPeriods.ts
Comment (haydencleary) [reply src/components/New/Filter/useDatepickerPeriods.ts:1]: d7156548f993f6e201dd40fad770b6918fe55e14

File: src/components/New/Field/Datepicker.tsx (hunk 1 above)
Comment (haydencleary) [reply src/components/New/Field/Datepicker.tsx]: d7156548f993f6e201dd40fad770b6918fe55e14

**Review by alexbeno — APPROVED** [review]
💯

**Pull request merged**